web/public/template.py

```diff
@@ -23,12 +23,15 @@ def place_market_order(side: str, ticker: str, quantity: float, price: float) ->
 class Strategy:
     """Template for a strategy."""
 
-    def __init__(self) -> None:
+    def __init__(self, holdings) -> None:
```

Comment:
Type hints pls

Reply:
I don't know what the type of holdings is

Reply:
@stevenewald steev we need halp

Reply:
remove it we dont need it